src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/AcknowledgeClaimApplicantNotificationHandlerTest.java

```diff
@@ -87,12 +95,141 @@ void shouldNotifyRespondentSolicitor_whenInvokedWithCcEvent() {
+            //solicitor 1 acknowledges claim,solicitor 2 already acknowledged
+            CaseData caseData = CaseDataBuilder.builder().atStateNotificationAcknowledged()
+                .addRespondent2(YES)
+                .respondent2SameLegalRepresentative(NO)
+                .respondent2(PartyBuilder.builder().individual().build())
+                .respondent1AcknowledgeNotificationDate(LocalDateTime.now())
+                .respondent2AcknowledgeNotificationDate(LocalDateTime.now().minusDays(2))
+                .respondent2ResponseDeadline(LocalDateTime.now().plusDays(14))
+                .respondent1ResponseDeadline(LocalDateTime.now().plusDays(14))
+                .build();
+            CallbackParams params = CallbackParamsBuilder.builder().of(ABOUT_TO_SUBMIT, caseData).request(
+                CallbackRequest.builder().eventId("NOTIFY_APPLICANT_SOLICITOR1_FOR_CLAIM_ACKNOWLEDGEMENT_CC").build())
+                .build();
+
+            handler.handle(params);
+
+            verify(notificationService).sendMail(
+                "respondentsolicitor@example.com",
+                "template-id",
+                getNotificationDataMap(caseData),
+                "acknowledge-claim-applicant-notification-000DC001"
+            );
+        }
+
+        @Test
+        void shouldNotifyRespondentSolicitor1_whenInvokedWithCcEvent() {
+            //solicitor 1 acknowledges claim,solicitor 2 not
+            CaseData caseData = CaseDataBuilder.builder().atStateNotificationAcknowledged()
+                .addRespondent2(YES)
+                .respondent2SameLegalRepresentative(NO)
+                .respondent2(PartyBuilder.builder().individual().build())
+                .respondent1AcknowledgeNotificationDate(LocalDateTime.now().plusDays(14))
+                .respondent2AcknowledgeNotificationDate(null)
+                .respondent1ResponseDeadline(LocalDateTime.now())
+                .respondent2ResponseDeadline(null)
+                .build();
+            CallbackParams params = CallbackParamsBuilder.builder().of(ABOUT_TO_SUBMIT, caseData).request(
+                CallbackRequest.builder().eventId("NOTIFY_APPLICANT_SOLICITOR1_FOR_CLAIM_ACKNOWLEDGEMENT_CC").build())
+                .build();
+
+            handler.handle(params);
+
+            verify(notificationService).sendMail(
+                "respondentsolicitor@example.com",
+                "template-id",
+                getNotificationDataMap(caseData),
+                "acknowledge-claim-applicant-notification-000DC001"
+            );
+        }
+
         @NotNull
         private Map<String, String> getNotificationDataMap(CaseData caseData) {
+            LocalDateTime responseDeadline = caseData.getRespondent1ResponseDeadline();
+            Party respondent = caseData.getRespondent1();
+            MultiPartyScenario multiPartyScenario = getMultiPartyScenario(caseData);
```

Comment:
This looks to be a duplication of the same statements written in the class as well. it would be better if you mark a TODO for now to remove this duplication and move it to a separate file